pkg/clustermanager/ssh_communicator.go

```diff
@@ -69,12 +72,14 @@ func (sshComm *SSHCommunicator) newSession(node Node) (*ssh.Session, *ssh.Client
 		User:            "root",
 		Auth:            []ssh.AuthMethod{ssh.PublicKeys(signer)},
 		HostKeyCallback: ssh.InsecureIgnoreHostKey(),
+		Timeout:         10 * time.Second,
 	}
 	var connection *ssh.Client
 	for try := 0; ; try++ {
 		connection, err = ssh.Dial("tcp", node.IPAddress+":22", config)
 		if err != nil {
-			//log.Printf("dial failed:%v", err)
+			log.Printf("dial failed:%v", err)
+			log.Printf("retrying..")
```

Comment:
We do not have parameter interpolation here, we should use `log.Print()` here.

PS: I'm not sure that we should log the retry operation

Reply:
Fixed the Printf() thing. But for the retrying, I think it is important because I got stuck with this for a long time without any debug info and that's how I ended up digging into the code. 
Eventually I found out that the error was related to old values in the hetzner-kube.config that were not cleaned up. So yeah, I think it is important to tell the dev that we are retrying.